pkg/ipfailover/keepalived/generator.go

```diff
@@ -165,8 +165,13 @@ func GenerateDeploymentConfig(name string, options *ipfailover.IPFailoverConfigC
 		return nil, err
 	}
 
+	labels := map[string]string{
+		"ipfailover": name,
```

Comment:
fwiw I was following the convention of how the router dc/template is labeled:

https://github.com/openshift/origin/blob/master/pkg/cmd/admin/router/router.go#L499

I don't care what convention we're using, but I'd like it to be reflected in the code.  If ipfailover uses name=, then the router should too.